tests/Transports.AspNetCore.Tests/NewtonsoftJsonTests.cs

```diff
@@ -41,64 +41,57 @@ public async Task Decodes_BigInteger()
-        }
-
-        [Fact]
-        public async Task Name_Matching_Not_Case_Sensitive()
-        {
-            var ret = await Deserialize(@"{""VARIABLES"":{""date"":""2015-12-22T10:10:10+03:00""}}");
-            ret.Single.Inputs["date"].ShouldBeOfType<string>().ShouldBe("2015-12-22T10:10:10+03:00");
+            ret.Single().Extensions.ShouldBeNull();
         }
 
         [Fact]
         public async Task Decodes_Multiple_Queries()
         {
             var ret = await Deserialize(@"[{""query"":""abc""},{""query"":""def""}]");
-            ret.Batch.Length.ShouldBe(2);
-            ret.Batch[0].Query.ShouldBe("abc");
-            ret.Batch[1].Query.ShouldBe("def");
+            ret.Length.ShouldBe(2);
+            ret[0].Query.ShouldBe("abc");
+            ret[1].Query.ShouldBe("def");
         }
 
         [Fact]
         public async Task Decodes_Nested_Dictionaries()
         {
             var ret = await Deserialize(@"{""variables"":{""a"":{""b"":""c""}},""extensions"":{""d"":{""e"":""f""}}}");
-            ret.Single.Inputs["a"].ShouldBeOfType<Dictionary<string, object>>()["b"].ShouldBe("c");
-            ret.Single.Extensions["d"].ShouldBeOfType<Dictionary<string, object>>()["e"].ShouldBe("f");
+            ret.Single().Variables["a"].ShouldBeOfType<Dictionary<string, object>>()["b"].ShouldBe("c");
+            ret.Single().Extensions["d"].ShouldBeOfType<Dictionary<string, object>>()["e"].ShouldBe("f");
         }
 
         [Fact]
         public async Task Decodes_Nested_Arrays()
         {
             var ret = await Deserialize(@"{""variables"":{""a"":[""b"",""c""]},""extensions"":{""d"":[""e"",""f""]}}");
-            ret.Single.Inputs["a"].ShouldBeOfType<List<object>>().ShouldBe(new[] { "b", "c" });
-            ret.Single.Extensions["d"].ShouldBeOfType<List<object>>().ShouldBe(new[] { "e", "f" });
+            ret.Single().Variables["a"].ShouldBeOfType<List<object>>().ShouldBe(new[] { "b", "c" });
+            ret.Single().Extensions["d"].ShouldBeOfType<List<object>>().ShouldBe(new[] { "e", "f" });
         }
 
-        private async Task<GraphQLRequestDeserializationResult> Deserialize(string jsonText)
+        private async Task<GraphQLRequest[]> Deserialize(string jsonText)
         {
             var jsonStream = new System.IO.MemoryStream(Encoding.UTF8.GetBytes(jsonText));
-            var httpRequest = new TestHttpRequest
+            var deserializer = new NewtonsoftJson.GraphQLSerializer(config =>
             {
-                Body = jsonStream
-            };
-            var deserializer = new NewtonsoftJson.GraphQLRequestDeserializer(_ => { });
-            return await deserializer.DeserializeFromJsonBodyAsync(httpRequest);
+                config.DateFormatHandling = Newtonsoft.Json.DateFormatHandling.IsoDateFormat;
+                config.DateParseHandling = Newtonsoft.Json.DateParseHandling.None;
+            });
+            return await deserializer.ReadAsync<GraphQLRequest[]>(jsonStream);
         }
```

Comment:
To use date parsing as it was before, `DateFormatHandling` and `DateParseHandling` must be set.

Should we change the above test to simply verify the serializer's default configuration and remove these lines here?

Reply:
I'm almost sure yes for `DateFormatHandling`. `DateParseHandling` is a PITA for many developers, the "main shame" of Newtonsoft.

> Should we change the above test to simply verify the serializer's default configuration and remove these lines here?

Yes, it will be better to ensure that the default configuration just works.